navigator_engine/common/bdg_message_extractor.py

```diff
@@ -0,0 +1,93 @@
+from navigator_engine.app import app
```

Comment:
I didn't think we were using this file anymore?  I thought we were parsing everything in the graph_loader and putting into our i18n sqlalchemy framework? If we can delete this file we should.

Reply:
It's been removed